benchmarks/100.webapps/110.dynamic-html/bun/function.js

```diff
@@ -0,0 +1,29 @@
+const Mustache = require('mustache'),
```

Comment:
Is the benchmark code very similar or identical to nodejs? I think that we might want to avoid duplication. I think having a separate directory for bun helps, but perhaps we could use a symlink or store a "delta" patch if there's a difference (the latter would require a separate step to copy nodejs + apply patch, but it's just few lines in Python)